### PR TITLE
Update model-selection.ipynb

### PR DESCRIPTION
**Pull request by JERRYenSHU503**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] [local build the book](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [ ] review the artifacts from local build
- [ ] [local build the slides](https://github.com/ocademy-ai/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [ ] Unit test if appliable

# Checklist:

- [ ] My code follows the [style guidelines](https://github.com/ocademy-ai/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have applied tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb [hunk 1]
```diff
@@ -49,7 +49,9 @@
     "\n",
```
Comment (Lola-jo) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <ul><li>Split into multiple chunks of markdown.</li><li>Why are the bolded headings so long, keep the headings short, and focus on the fact that they can be emphasized in other ways.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (Lola-jo) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>Same, split the markdown cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (Lola-jo) [reply open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>The problem hasn't been solved. The aesthetics are terrible.</p>

File: open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb [hunk 2]
```diff
@@ -49,7 +49,13 @@
...
```
Comment (Lola-jo) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>There are too many pictures attached, add notes as appropriate, or put the pictures in the right place for instruction</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (Lola-jo) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>Same. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

File: images/model-selection/confusion_matrix.jpg
Comment (Lola-jo) [inline images/model-selection/confusion_matrix.jpg:1]: Confusion matrix results are easy to implement in code, and will be more graphic, there is no need to use other people's images!

File: open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb (hunk 1 above)
Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>Split into multiple chunks of markdown.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>There are spelling errors, please check carefully.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p> </p><p>Like Accuracy，Precision and Recall and so on , they can be displayed in bold to ensure readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>Same, like Accuracy，Precision and Recall and so on , they can be displayed in bold to ensure readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>Split into multiple chunks of markdown.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>Same, like Rule of thumb , Cross-validation and so on , they can be displayed in bold to ensure readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>There is a redundant change line before Metrics, remove it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>No need to add cell tag.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>Same, remove the cell tag.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>There is a redundant change line before Capacity, remove it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>There is a redundant change line before Early Stopping, remove it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (Nicole-ying) [inline open-machine-learning-jupyter-book/ml-advanced/model-selection.ipynb:49]: <p>There is a redundant change line before Conclusions, remove it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Comment by Nicole-ying** [comment]
Since the image of Confusion Matrix is ​​no longer needed, please delete this file.

**Review by Nicole-ying — APPROVED** [review]
OK

**Pull request merged**